common/economic_plans/03_post_science.txt

```diff
@@ -0,0 +1,259 @@
+#If my calculations are correct, repetable technologies are very, very inefficient
+#Thus, once AI have researched everything useful they will switch to fully alloyed economy
+
+
+# Middle ----------------------------------------
+
+middle_imperial_plan = {
+
+	income = {
+		energy = 300
+		minerals = 500
+		alloys = 500
+		unity = 100
+
```

Comment:
We'll want to revisit these as well, if I'm understanding how this works properly, this might not work as intended considering we've removed exotic gases, motes, and crystals.